rero_ils/modules/documents/mappings/v7/documents/document-v0.0.1.json

```diff
@@ -90,56 +156,36 @@
                 "properties": {
                   "value": {
                     "type": "text",
-                    "index": false,
-                    "copy_to": "title._search"
+                    "fields": {
+                      "eng": {
+                        "type": "text",
+                        "analyzer": "english"
+                      },
+                      "fre": {
+                        "type": "text",
+                        "analyzer": "french"
+                      },
+                      "ger": {
+                        "type": "text",
+                        "analyzer": "german"
+                      },
+                      "ita": {
+                        "type": "text",
+                        "analyzer": "italian"
+                      }
+                    }
+                  },
+                  "language": {
+                    "type": "keyword"
                   }
                 }
               }
             }
           },
           "_text": {
             "type": "text",
-            "index": false,
```

Comment:
As discussed, I think we should keep indexing all "_text" fields but without the language analyzers. This would allow phrase searches across multiple subfields to keep working.